recipe_scrapers/nihhealthyeating.py

```diff
@@ -23,9 +23,7 @@ def total_time(self):
         if time_table is None:
             raise ElementNotFoundInHtml("Table with times was not found.")
 
-        return sum(
-            [get_minutes(td) for td in time_table.find_all("td") if get_minutes(td)]
-        )
+        return sum(get_minutes(td) or 0 for td in time_table.find_all("td")) or None
```

Comment:
Note: I applied one further change here, to return `None` instead of zero if the sum-of-times evalues to zero.

Reply:
Dropping this change; todo: re-apply this separately later.